EAD-Session4/src/com/metacube/Parking/dao/AddVehicleDAO.java

```diff
@@ -0,0 +1,84 @@
+/**
+* This Servlet is a DAO class for Adding Vehical 
+* @author Abhishek Bishnoi
+* @since Aug 29,2019
+*/
+package com.metacube.Parking.dao;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import com.metacube.Parking.model.Employee;
+import com.metacube.Parking.model.Vehical;
+import com.metacube.Parking.utility.Connector;
+import com.metacube.Parking.utility.Query;
+
+public class AddVehicleDAO {
+	/**
+	* @param vehicle Object Of Vehicle
+	* @return boolean value as success or not
+	*/
+	public boolean addEmployee(Vehical vehicle){
+		Connector connector = new Connector();
+		String query = Query.addVehicle();
+		boolean result = true;
+		PreparedStatement prepared = null;
+		try(Connection connect = connector.getConnection();
+				PreparedStatement preparedStatement = connect.prepareStatement(query)){
+			connect.setAutoCommit(false);
+			preparedStatement.setString(1, vehicle.getEmployeeEmail());
+			preparedStatement.setString(2, vehicle.getVehicleType());
+			preparedStatement.setString(3, vehicle.getVehicleNumber());
+			preparedStatement.setString(4, vehicle.getDescription());
+			preparedStatement.execute();
+			query = Query.updateIsVehicle();
+			prepared = connect.prepareStatement(query);
+			prepared.setInt(1, 1);
+			prepared.setString(2, vehicle.getEmployeeEmail());
+			int i = prepared.executeUpdate();
+			connect.commit();
+			
+		} catch (ClassNotFoundException | SQLException | IOException e) {
+			e.printStackTrace();
+			return false;
+		}finally{
+			try {
+				prepared.close();
```

Comment:
No need of close() function if we are using IO like:
try(Connection connect = connector.getConnection();
				PreparedStatement preparedStatement = connect.prepareStatement(query))